block/models.py

```diff
@@ -1363,6 +1365,30 @@ def is_external(self):
         return bool(self.link_type == self.URL_EXTERNAL)
 
     @property
+    def blocks_used(self):
+        collector = NestedObjects(using='default')
```

Comment:
@mdinsmore would you mind adding a short `docstring` to this describing what you are doing with `NestedObjects` please?  I can't see Django documentation for the class.
Thank you